src/main/java/seedu/address/model/expenses/Money.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.model.expenses;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+import seedu.address.model.expenses.exceptions.MoneyLimitExceededException;
+
+/**
+ * Class to represent monetary values.
+ * Ensures that prices keyed in for Expense objects are valid.
+ * Range of dollars limited to +/- Integer.MAX_VALUE.
+ * Range does not include Integer.MIN_VALUE to prevent errors when changing signs.
+ */
+public class Money {
+
+    private int dollars;
+    private int cents;
+    //v2.0: implement exchange rates
+
+    public Money(int dollars, int cents) {
```

Comment:
Do you intend to let other parts of the app use this constructor? If not, set it as private.

Reply:
Having multiple ways to construct an object can open up more testing issues down the road.